widgetastic_manageiq/__init__.py

```diff
@@ -868,7 +868,7 @@ def click_at(self, item_name):
 
         return self.list(item_name).list_click()
 
-    def is_displayed(self, (item_name)):
```

Comment:
**REQUIRED:** Can you explain this one a little please?

Reply:
the syntax is equivalent on python2 as it allowed tuple unpacking in function arguments,
its a syntaxerror on python3